### PR TITLE
add a note about php extensions installation on XAMPP and WAMP

### PR DESCRIPTION
**Pull request by cake17**
Summarize all comments made in #1687 to explain how to install php extensions in xampp and wamp installations

### CODE REVIEW

File: en/installation.rst [hunk 1]
```diff
@@ -16,6 +16,21 @@ Requirements
...
+
+    In both XAMPP and WAMP, mcrypt and mbstring extensions are working by default.
+
+    In XAMPP, intl extension is included but you have to uncomment
+    extension=php_intl.dll in php.ini and restart the server through the XAMPP
+    Control Panel.
+
+    In WAMP, the intl extension is "activated" by default but not working.
+    To make it work you have to go to php folder (by default)
+    C:\wamp\bin\php\php5.5.12, copy all the files that looks like icu***.dll
```
Comment (dereuromark) [inline en/installation.rst]: I wonder if one should use `php{version}` here to be not version specific.

Comment (cake17) [inline en/installation.rst]: yes I was wondering how to format this, I'll add a commit with yours.

Comment (dereuromark) [inline en/installation.rst]: Also, the paths should maybe be in the `double backticks`, same as filenames etc.

**Comment by cake17** [comment]
I can add these comments in another place if you want ?

**Pull request merged**